### ISSUE TITLE
ensure consistent `Rune`ing in examples

### ISSUE THREAD

**Issue by nythrox** [paritytech/capi#692]
In `examples/multisig_transfer`, the way we execute the program is by awaiting multiple `run` calls:

```ts

// Submit a proposal to dispatch the call
await multisig
  .ratify({ call, sender: alice.address })
  // [..]
  .run()

// Check if the call has been proposed
console.log("Is proposed?:", await multisig.isProposed(call.hash).run())

// Send a non-executing approval
await multisig
  .approve({
    callHash: call.hash,
    sender: bob.address,
  })
  // [..]
  .run()

// Check for existing approval(s)
console.log(
  "Existing approvals:",
  await multisig
    .proposal(call.hash)
  // [..]
    .run(),
)

// Send the executing (final) approval
await multisig
  .ratify({ call, sender: charlie.address })
  // [..]
  .run()
```

In `examples/virtual_multisig` we have a single `run` call that contains the whole sequence of the program:
```ts
await Rune
  .chain(() => vMultisig)
  .chain(() => fundStash)
  .chain(() => System.Account.entry([dave.publicKey]).dbg("Dave Balance Before:"))
  .chain(() => bobTx)
  .chain(() => charlieTx)
  .chain(() => System.Account.entry([dave.publicKey]).dbg("Dave Balance After:"))
  .run()
```

Should the first example be updated to chain the program together and then `run` it? Or do we encourage both ways

**Comment by harrysolovay** [comment]
Great question––I'd say we prefer the first, as it can be better-optimized and eventually serialized into other formats. Shall we rename this issue to something like this?: "ensure consistent `Rune`ing in examples"

**Comment by harrysolovay** [comment]
Addressed by #826
